slasher/detection/attestations/spanner.go

```diff
@@ -163,7 +163,8 @@ func (s *SpanDetector) updateMinSpan(source uint64, target uint64, valIdx uint64
 	if source < 1 {
 		return
 	}
-	for epoch := source - 1; epoch >= 0; epoch-- {
+	for epochInt := int64(source - 1); epochInt >= 0; epochInt-- {
```

Comment:
Why int64 and not uint64

Reply:
This is a bug I introduced when I tried to fix spans on epoch 0, since it's a for loop going to 0, when we decrement after 0 it underflows and becomes max uint64. Changing it to an int lets it become negative.

Reply:
I think shay mentioned this issue, wouldnt this be fixed if we just make the conditional epochInt > 0 instead of epochInt >= 0?

Reply:
Then it wouldn't process 0 no? It's what we were doing previously and spans for epoch 0 weren't getting marked.